### PR TITLE
Restructured the directory for all additional resources like Data/ 

### PR DESCRIPTION
**Pull request by ThorstenBux**
Data2/ to /share/artoolkit-examples and share/artoolkit-utils

### CODE REVIEW

File: examples/nftBook/nftBook.c [hunk 1]
```diff
@@ -158,12 +158,12 @@ int main(int argc, char** argv)
...
     char   *vconf = NULL;
-    char    cparaDefault[] = "Data2/camera_para.dat";
+    char    cparaDefault[] = "../share/artoolkit-examples/Data2/camera_para.dat";
     char   *cpara = NULL;
     int     i;
     int     gotTwoPartOption;
-    const char markerConfigDataFilename[] = "Data2/markers.dat";
-	const char objectDataFilename[] = "Data2/objects.dat";
+    const char markerConfigDataFilename[] = "../share/artoolkit-examples/Data2/markers.dat";
+	const char objectDataFilename[] = "../share/artoolkit-examples/Data2/objects.dat";
```
Comment (BoiseBaked) [inline examples/nftBook/nftBook.c]: Looks like a tab character.

File: examples/optical/optical.c [hunk 2]
```diff
@@ -167,14 +167,14 @@ int main(int argc, char** argv)
...
     char   *vconf = NULL;
-    char    cparaDefault[] = "Data/camera_para.dat";
+    char    cparaDefault[] = "../share/artoolkit-examples/Data/camera_para.dat";
     char   *cpara = NULL;
     int     i;
     int     gotTwoPartOption;
-    const char markerConfigDataFilename[] = "Data/markers.dat";
-	const char objectDataFilename[] = "Data/objects.dat";
+    const char markerConfigDataFilename[] = "../share/artoolkit-examples/Data/markers.dat";
+	const char objectDataFilename[] = "../share/artoolkit-examples/Data/objects.dat";
```
Comment (BoiseBaked) [inline examples/optical/optical.c]: Remember, change all tab characters to 4 spaces.

File: share/artoolkit-examples/Wrl/snoman.wrl [hunk 3]
```diff
@@ -1,244 +1,244 @@
...
-ROUTE Lift.value_changed TO Tipper.set_spine 
-ROUTE Tick.fraction_changed TO Doff.set_fraction
-ROUTE Doff.value_changed TO Hat.set_translation  
-
-WorldInfo {
-   title "Arcadia"
-   info [ "(C)1997 modeled by Kiwano (Andrew Reitemeyer) kiwano@t-online.de"]
-          
-          }
+#VRML V2.0 utf8
```
Comment (BoiseBaked) [inline share/artoolkit-examples/Wrl/snoman.wrl]: Why is this file flagged as changed?

File: share/artoolkit-utils/Data/hiro.patt [hunk 4]
```diff
@@ -0,0 +1,196 @@
+ 234 235 240 233 240 234 240 235 240 237 240 238 240 240 240 232
```
Comment (BoiseBaked) [inline share/artoolkit-utils/Data/hiro.patt]: Have to figure out why hiro.patt is always indicated as git merge-able.

**Comment by BoiseBaked** [comment]
Thor, it all looks good. Just a few tab characters where there is supposed to be space characters but don't go crazy fixing all files per this issues. Just fix a few files with the most changes.

Also, GitHub indicates it can do the PR merge due to merge conflicts. So you have to rebase or merge against your branch and master and resolve the the merge conflicts. And then re-push (which may require you to close the current PR, delete and re-create your remote (origin) branch first).

File: share/artoolkit-examples/Wrl/snoman.wrl (hunk 3 above)
Comment (ThorstenBux) [inline share/artoolkit-examples/Wrl/snoman.wrl]: somehow git did not recognise that it was moved and therefor marks it as deleted and added.

File: examples/nftBook/nftBook.c (hunk 1 above)
Comment (ThorstenBux) [inline examples/nftBook/nftBook.c]: done

File: examples/optical/optical.c (hunk 2 above)
Comment (ThorstenBux) [inline examples/optical/optical.c]: done

**Pull request merged**